os/net/blemgr/bledev_mgr_client.c

```diff
@@ -66,7 +66,7 @@ static void bledrv_device_connected_cb(trble_device_connected *dev)
 
 static void bledrv_operation_notification_cb(trble_operation_handle *handle, trble_data *read_result)
 {
-	uint32_t size = sizeof(trble_conn_handle) + sizeof(trble_attr_handle) + sizeof(read_result->length) + read_result->length;
+	int32_t size = sizeof(trble_conn_handle) + sizeof(trble_attr_handle) + sizeof(read_result->length) + read_result->length;
```

Comment:
Is it okay not to check `read_result` is NULL or not?

Reply:
'read_result' is guranteed not to be NULL in BLE Stack layer.